test/runnable/testdstress.d

```diff
@@ -1,6 +1,6 @@
 // PERMUTE_ARGS:
 
-module dstress.run.module_01;
+module run.module_01;
```

Comment:
Why was this change necessary?

Reply:
I'm not entirely sure why, but for `dstress.run.module_01` you would get that `dstress.run` is not accessible from the current module; that is probably because there is no directory structure that matches the module declaration.

Reply:
But the folder `run` doesn't exist either, does it?

Reply:
actually its runnable

Reply:
That module declaration looks like a leftover from when code was copied into the `test/runnable` directory.  Can't we just get rid of it?